src/js/src/components/SearchFacetOptions.vue

```diff
@@ -78,10 +78,16 @@ export default {
       this.addSpecificRequestedFacets({facet:facetArea, query:structuredQuery, appliedFacets:appliedFacets})
     },
     applyFacet(facetCategory, facet) {
+      let facetAllreadyApplied = false
       let newFacet = '&fq=' + facetCategory + ':"' + facet + '"'
-      this.updateSolrSettingOffset(0)
-      this.addToSearchAppliedFacets(newFacet)
-      this.$_pushSearchHistory('Search', this.query, this.searchAppliedFacets, this.solrSettings)
+      this.searchAppliedFacets.forEach((facet) => {
+        facet === newFacet ? facetAllreadyApplied = true : null
+      })
+      if(!facetAllreadyApplied === true) {
```

Comment:
Could shorten this to `if(!facetAllreadyApplied) {`

Reply:
Aaah! Yes, i'll fix that!